aptos-move/block-executor/src/executor.rs

```diff
@@ -62,6 +63,7 @@ where
         executor: &E,
         base_view: &S,
     ) -> SchedulerTask<'a> {
+        let _timer = TASK_EXECUTE_SECONDS.start_timer();
```

Comment:
Can we also have a cpu timer here?

https://docs.rs/cpu-time/latest/cpu_time/

Reply:
Let's add this as a follow up - want to experiment with CPU timers to see how accurate they are and understand the overhead.